tests/scanner/scanners/ke_jmespath_scanner_test.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+# Copyright 2018 Ross Vandegrift. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""KE jmespath Rule Scanner Tests."""
```

Comment:
ditto, can you please leave `jmespath` out of the name and call this file to be `ke_scanner_test.py?